h/tasks/moderation.py

```diff
@@ -0,0 +1,96 @@
+from datetime import datetime
+
+from sqlalchemy import select
+
+from h.models import ModerationLog, ModerationStatus
+from h.models.notification import NotificationType
+from h.tasks.celery import celery, get_task_logger
+from h.services.notification import NotificationService
+
+log = get_task_logger(__name__)
+
+
+@celery.task
+def send_moderation_email(annotation_id, moderation_datetime_iso: str):
+    db = celery.request.db
+
+    print("send_moderation_email", annotation_id, moderation_datetime_iso)
+    moderation_datetime = datetime.fromisoformat(moderation_datetime_iso)
+
+    moderation_log = db.scalars(
+        select(ModerationLog)
+        .where(
+            ModerationLog.annotation_id == annotation_id,
+            ModerationLog.created >= moderation_datetime,
+            ModerationLog.notification_id.is_(None),
+        )
+        .order_by(ModerationLog.created.asc())
+        .with_for_update(skip_locked=True)
+    ).all()
+
+    if not moderation_log:
```

Comment:
This case is when no moderation status changes have been found. Not sure in which scenario.